### PR TITLE
Add tests to check up status for all components

### PR DESCRIPTION
**Pull request by nhudson**
#### What this PR does / why we need it

Helm test to check if components of the stack are actually up and running

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes # https://github.com/timescale/tobs/issues/324

#### Checklist

- [x] Chart Version bumped
- [x] [CLA signed](https://cla-assistant.io/timescale/tobs)

### CODE REVIEW

**Review by paulfantom — COMMENTED** [review]
Awesome work! :100: Let's fix those CI issues and merge it as soon as possible.

File: chart/templates/tests/test-up-timescaledb.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    # Setup query to prometheus
+    RELEASE="{{ .Release.Name }}"
+    NAMESPACE="{{ .Release.Namespace }}"
+
+    PROM_QUERY_URL="http://${RELEASE}-promscale.${NAMESPACE}.svc:9201/api/v1/query?"
+
+    function query() {
+      local query
+      query=$(cat <<-EOF
+    query=up==0
```
Comment (paulfantom) [inline chart/templates/tests/test-up-timescaledb.yaml:78]: Do you think we could have here also a test for `prometheus_build_info` metric existence or do you think we should create a separate PR for that?

**Comment by nhudson** [comment]
After a ton of debugging it looks like kind, sometimes doesn't allow scrapes to the following pods once everything is installed 

```
kube-proxy
kube-controller-manager
kube-scheduler
etcd
```

This causes the test to fail due to the `up` status being `0` for these 4 pods.  Checking pods these look to be healthy and as far as I can tell the cluster is running with no issues.

**Comment by nhudson** [comment]
Possible short term solution might be to just disable the rules that would scrape these endpoints 

```
kube-prometheus-stack:
  defaultRules:
    rules:
      etcd: false
      kubeControllerManager: false
      kubeProxy: false
      kubeScheduler: false
```

**Comment by nhudson** [comment]
Also since it takes some time for Promscale to come online, the `up` metric does not get updated by the time the test runs.  This will cause the test to fail as well.

**Comment by paulfantom** [comment]
Let's disable scrape configs and rules for endpoints listed in https://github.com/timescale/tobs/pull/564#issuecomment-1231725609. 

As for promscale, I was under impression that helm test are run after release is deployed. In our case we are waiting until promscale is in Ready state. This means the `up` metric should have a value of `1` in next scrape after promscale is Ready. For now let's put just a `sleep 30` before running the query. In the future I think we should move all our tests into some golang application and run those queries with a backoff mechanism to reduce test flakiness.

**Review by paulfantom — APPROVED** [review]

File: chart/templates/tests/test-up-timescaledb.yaml (hunk 1 above)
Comment (nhudson) [reply chart/templates/tests/test-up-timescaledb.yaml:78]: Just saw I never answered this.  I was going to suggest just creating a new PR for this.  I'll go ahead and create the issue.
